packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -622,10 +622,6 @@ export function markRootPinged(root: FiberRoot, pingedLanes: Lanes) {
   root.pingedLanes |= root.suspendedLanes & pingedLanes;
 }
 
-export function markRootMutableRead(root: FiberRoot, updateLane: Lane) {
-  root.mutableReadLanes |= updateLane & root.pendingLanes;
```

Comment:
Can you grep for `mutableReadLanes`, I think it can be removed from the fiber too.